scripts/deployment/_push-assets-to-cdn

```diff
@@ -167,7 +167,14 @@ for mimetype in mimetypes_to_process:
     cp -Z -n -r . "{BUCKET}"
     '''
 
-  subprocess.run(gsutil_command, shell=True, cwd=mimetype_subfolder)
+  result = subprocess.run(gsutil_command, shell=True, cwd=mimetype_subfolder, capture_output=True)
+  if result.stderr and result.returncode != 0:
+    raise subprocess.CalledProcessError(
```

Comment:
Maybe `raise` is the wrong thing here - I don't know python.

Reply:
why non-empty stderr and non-zero returncode?

I would print an error and then use `system.exit`

Reply:
We could suppress the non-error messages getting to stderr with `-q`, but those messages are useful for local debugging https://github.com/GoogleCloudPlatform/gsutil/issues/116#issuecomment-13929972 so I think this is best. Adding a comment.